src/GeneralStateTestsFiller/stEWASMTests/selfDestructExistingAccountFiller.yml

```diff
@@ -0,0 +1,65 @@
+# selfDestruct existing account
+---
+selfDestructExistingAccount:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: ''
+      nonce: '0'
+      storage: {}
+    b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      balance: '100000000000'
+      code: |
+        (module
+          (import "ethereum" "selfDestruct" (func $selfDestruct (param i32)))
+          (memory 1)
+          (data (i32.const 0) "\80\75\8d\9f\09\8c\e0\89\9d\c0\70\1f\9d\ad\fc\f4\ad\45\8d\c8")
```

Comment:
I think the beneficiary should be the `a94...0b` address as it is executing the transaction and the hackmd says it should self-destruct back into the original sender.